ui/page/tickets/activity_page.go

```diff
@@ -103,8 +101,16 @@ func (pg *ActivityPage) Layout(gtx layout.Context) layout.Dimensions {
 							})
 						})
 					}),
-					layout.Stacked(func(gtx C) D {
-						return pg.dropDowns(gtx)
+					layout.Expanded(func(gtx C) D {
+						return pg.walletDropDown.Layout(gtx, 0)
+					}),
+					layout.Expanded(func(gtx C) D {
+						post := float32(gtx.Constraints.Max.X - 150)
+						return pg.orderDropDown.Layout(gtx, post)
+					}),
+					layout.Expanded(func(gtx C) D {
+						post := float32(gtx.Constraints.Max.X - 315)
+						return pg.ticketTypeDropDown.Layout(gtx, post)
```

Comment:
So doing troubleshooting, this seems to be where the issue is coming from.

Changing the values as suggested below, give me the screenshots below.
![image](https://user-images.githubusercontent.com/27733432/132838330-2cd80f51-5392-4a09-995b-cd852760a527.png)

```suggestion
					layout.Expanded(func(gtx C) D {
						return pg.walletDropDown.Layout(gtx, 0)
					}),
					layout.Expanded(func(gtx C) D {
						post := float32(gtx.Constraints.Max.X - 450)
						return pg.orderDropDown.Layout(gtx, post)
					}),
					layout.Expanded(func(gtx C) D {
						post := float32(gtx.Constraints.Max.X - 300)
						return pg.ticketTypeDropDown.Layout(gtx, post)
```

I think the issue here has to do with the screen size of the machine. you will need to use `gtx.Px()` for positioning and scaling.